vpr/src/route/rr_graph_reader.cpp

```diff
@@ -363,6 +380,21 @@ void process_nodes(pugi::xml_node parent, const pugiutil::loc_data & loc_data) {
         //clear each node edge
         node.set_num_edges(0);
 
+        //  <metadata>
+        //    <meta name='grid_prefix' x_offset="1" y_offset="0" z_offset="0">CLBLL_L_</meta>
+        //  </metadata>
+        auto metadata = get_single_child(rr_node, "metadata", loc_data, OPTIONAL);
+        if (metadata) {
+            auto rr_node_meta = get_first_child(metadata, "meta", loc_data);
+            while (rr_node_meta) {
+                auto key = get_attribute(rr_node_meta, "name", loc_data).as_string();
+
+                vpr::add_node_metadata(inode, key, rr_node_meta.child_value());
```

Comment:
add_rr_node_metadata would be a better name (we have lots of different kinds of nodes).